packages/tools/devtools/devtools-core/src/data-visualization/DefaultVisualizers.ts

```diff
@@ -47,6 +48,7 @@ export const visualizeSharedCell: VisualizeSharedObject = async (
 		children: { data: renderedData },
 		typeMetadata: "SharedCell",
 		nodeKind: VisualNodeKind.FluidTreeNode,
+		editProps: { editTypes: [EditType.string, EditType.number, EditType.number] },
```

Comment:
Since we don't yet have an editor for SharedCell, we should probably remove this for the time being.